source/time_stepping/convection_time_step.cc

```diff
@@ -52,10 +77,34 @@ namespace aspect
             fe_values[this->introspection().extractors.velocities].get_function_values (this->get_solution(),
                                                                                         velocity_values);
 
+            if (consider_darcy_timestep == true)
+              {
+                in.reinit(fe_values, cell, this->introspection(), this->get_solution());
+                this->get_material_model().create_additional_named_outputs(out);
+                this->get_material_model().evaluate(in, out);
+                this->get_material_model().fill_additional_material_model_inputs(in, this->get_solution(), fe_values, this->introspection());
```

Comment:
This line would need to happen before the call to `evaluate`, right?

Reply:
I just tried putting the line:

 `this->get_material_model().fill_additional_material_model_inputs(in, this->get_solution(), fe_values, this->introspection());`

 before `evalute`, after `evalute`, and not including at all, and it seems to give the same solution for all 3 cases so I'll remove it and see if it causes any of the other tests to fail.

Reply:
the line:

`this->get_material_model().create_additional_named_outputs(out);`

also doesn't seem to be required, so I'll remove this line too.